Source/WebKit/UIProcess/RemoteLayerTree/cocoa/RemoteLayerTreeLayers.mm

```diff
@@ -57,7 +57,6 @@ - (void)_setWKContents:(id)contents withDisplayList:(WebKit::CGDisplayList&&)dis
 
     if (replay) {
         _displayListDataForTesting = data;
-        ASSERT(displayList.ports.isEmpty());
```

Comment:
This is fine, since now we only have takeSurfaces(), no way to peek at them. Not an important assertion, anyway.